src/targets/resources/tree.js

```diff
@@ -9,10 +13,45 @@ export class Tree extends Resource {
     constructor(data) {
         data.skill = "woodcutting";
         super(data);
+        this.requiredWCLevel = data.requiredWCLevel;
     }
 
     isClickable() {
         let curWeapon = this.scene.dashboard.equipment.equipment.WEAPON;
-        return curWeapon.item == "Axe";
+        let inventory = this.scene.dashboard.inventory;
```

Comment:
I think this function should be generalized and moved into the resource class because it's shared between woodcutting and mining and if we add another resource later it will also require something similar. As far as I can tell the only thing you would need to do is pass in "Axe" and "woodcutting". You might also need to change`requiredWCLevel` into:
```
requiredLevels = {
    attack: 5,
    woodcutting: 6
}
```

Or whatever is easiest

Reply:
updated! https://github.com/EthanVieira/rs-clicker/pull/57/commits/8048b8819e2c3b3ec52cc513a0ede33b9f57569a